src/capture/pcap_service.c

```diff
@@ -223,6 +220,17 @@ int dump_file_pcap(struct pcap_context *ctx, char *file_path,
   return 0;
 }
 
+int inject_pcap(struct pcap_context *ctx, uint8_t *packet, size_t size) {
+  int sent;
+
+  if ((sent = pcap_inject(ctx->pd, packet, size)) < 0) {
```

Comment:
We should always initialise variables at declaration if we can. This follows the Google style guide, so we can call ourselves Google-C++-engineers https://google.github.io/styleguide/cppguide.html#Local_Variables :wink: 
```suggestion
  int sent = pcap_inject(ctx->pd, packet, size);

  if (sent < 0) {
```